src/components/toasts/TestnetToast.js

```diff
@@ -1,21 +1,24 @@
 import React, { useEffect, useState } from 'react';
-import networkTypes from '../../helpers/networkTypes';
 import { Icon } from '../icons';
 import { Nbsp, Text } from '../text';
 import Toast from './Toast';
 import { isHardHat } from '@/handlers/web3';
 import { useInternetStatus } from '@/hooks';
-import { getNetworkObj } from '@/networks';
+import { getNetworkObject } from '@/networks';
+import { ChainId } from '@/networks/types';
+import { useConnectedToHardhatStore } from '@/state/connectedToHardhat';
 
-const TestnetToast = ({ network, web3Provider }) => {
+const TestnetToast = ({ chainId }) => {
+  const { connectedToHardhat } = useConnectedToHardhatStore();
   const isConnected = useInternetStatus();
-  const providerUrl = web3Provider?.connection?.url;
-  const { name, colors: networkColors } = getNetworkObj(network);
-  const [visible, setVisible] = useState(!network === networkTypes.mainnet);
+  const { name, colors: networkColors } = getNetworkObject({ chainId });
+  const [visible, setVisible] = useState(true);
```

Comment:
any reason not to keep the default state to `chainId !== ChainId.mainnet` or to `false` here?

Reply:
nope, i'll change it back